client/src/App.js

```diff
@@ -41,28 +54,60 @@ export default class App extends Component {
       .finally(() => this.setState({ loading: false }));
   }
 
+  onChange = (event, { newValue, method }) => {
+    this.setState({
+      country: newValue
+    });
+  };
+
+  getSuggestionValue = suggestion => {
+    return suggestion.id;
+  };
+
+  renderSuggestion = suggestion => {
+    return suggestion.id;
+  };
+
+  onSuggestionsFetchRequested = ({ value }) => {
+    this.setState({
+      suggestions: getSuggestions(value)
+    });
+  };
+
+  onSuggestionsClearRequested = () => {
+    this.setState({
+      suggestions: []
+    });
+  };
+
   componentDidMount() {
     this.getCovidInformation();
   }
 
   render() {
+    const { country, suggestions } = this.state;
+   
     return (
       <div className="App">
         <h1 className="mb5">Covid - 19 Live Updates</h1>
         <div className="flex-center wrap mb5">
-          <input
-            type="text"
-            placeholder="Enter Country Name (Eg: US)"
-            onChange={this.handleInputChange}
-            onKeyDown={this.handleEnter}
-            value={this.state.country}
-            className="Search-Box"
+          <Autosuggest
+            suggestions={suggestions}
+            onSuggestionsFetchRequested={this.onSuggestionsFetchRequested}
+            onSuggestionsClearRequested={this.onSuggestionsClearRequested}
+            getSuggestionValue={this.getSuggestionValue}
+            renderSuggestion={this.renderSuggestion}
+            onSuggestionSelected={this.onSuggestionSelected}
+            inputProps={{
+              value: country,
+              onChange: (e, { newValue }) =>
+                this.setState({ country: newValue })
+            }}
```

Comment:
@IamSaquib add `onKeyDown: this.handleEnter` to handle the enter key press instead of only search click